init.lua

```diff
@@ -6,7 +6,7 @@ local mq = require 'mq'
 local ImGui = require 'ImGui'
 local actors = require 'actors'
 local settings = require 'xtxsettings'
-
+local theme = require('themes')
```

Comment:
this line can be removed my bad. 
i decided to play nice and put the functions and tables into the settings table.